src/documentation/templates/modules/content/icon-page.html

```diff
@@ -1,6 +1,10 @@
 <div class="docs-IconList clearfix">
```

Comment:
Hey @mikewheaton , this was previously hard coded HTML.  I changed it in commit 31ba6c6 to load the the json data and generate the page with handlebars